### PR TITLE
fix(CORE/Raid) Save correct data to DB after killing Ragnaros (issue #2464)

### PR DESCRIPTION
**Pull request by Bogir**
**Save correct data to DB after killing Ragnaros**

##### ISSUES ADDRESSED:
Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2464

##### TESTS PERFORMED:
After killing Ragnaross at the end of instance Sulfuras fall in his correct position and server writes correct data to DB about Ragnaross status ("3")

##### HOW TO TEST THE CHANGES:
Turn on GM-mode
Go to Blackrock Mountain -> Molten Core
Kill all bosses including Ragnaross.

Open table _characters_.**instance**.
You must have correct data 
изображение
It was incorrect: M C 3 3 3 3 3 3 3 3 3 **1**

##### Target branch(es):
- [x] Master

 

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/server/game/AI/ScriptedAI/ScriptedCreature.cpp [hunk 1]
```diff
@@ -517,6 +517,10 @@ void BossAI::_JustDied()
     {
         instance->SetBossState(_bossId, DONE);
         instance->SaveToDB();
+		if (me->GetGUID() == 11121) {	//Boss Ragnaross
```
Comment (Viste) [inline src/server/game/AI/ScriptedAI/ScriptedCreature.cpp]: tabs -> space
исправь идент на 4 пробела!

Comment (Viste) [reply src/server/game/AI/ScriptedAI/ScriptedCreature.cpp]: ну т.е. твой if должен быть на уровне instance->SaveToDB(); итд

Comment (Viste) [reply src/server/game/AI/ScriptedAI/ScriptedCreature.cpp]: { не после if, а на отельной строке, новой как везде

Comment (Bogir) [reply src/server/game/AI/ScriptedAI/ScriptedCreature.cpp]: Spaces are fixed :D

File: src/server/game/AI/ScriptedAI/ScriptedCreature.cpp [hunk 2]
```diff
@@ -517,6 +517,8 @@ void BossAI::_JustDied()
     {
         instance->SetBossState(_bossId, DONE);
         instance->SaveToDB();
+        if (me->GetGUID() == 11121) //Boss Ragnaross
```
Comment (Stoabrogga) [inline src/server/game/AI/ScriptedAI/ScriptedCreature.cpp]: Sorry, but this is the wrong place for custom content. I think to solve the issue it suffices to call "BossAI::JustDied" in "boss_ragnaros.cpp":
```cpp
void JustDied(Unit* killer) override
{
    BossAI::JustDied(killer);
    me->SetFacingTo(DEATH_ORIENTATION);
}
```

**Comment by Stoabrogga** [comment]
No need to close the PR. I re-added the missing file and tested the PR, works fine now, the data is now correctly saved into the DB.

**Pull request merged**